### ISSUE TITLE
remove styles

### ISSUE THREAD

**Issue by samburgers** [dalphyx/vue-headroom#13]
Hi there, any chance for the option to remove any inline styles, and just use the classes for CSS changes.

Cheers!

**Comment by jdvivar** [comment]
Yes, you can do that yourself. You need to redefine the `style` function from this line:
https://github.com/dalphyx/vue-headroom/blob/master/src/headroom.vue#L123

That computed function is only used in the style property:
https://github.com/dalphyx/vue-headroom/blob/master/src/headroom.vue#L2

When importing, just redefine the function to return an empty string instead:
```js
import headroom from 'vue-headroom'
headroom.computed.style = () => ''
```

I hope it helps!

**Comment by missmatsuko** [comment]
Can this be documented? I thought this plugin would be more like the original headroom.js, but the inline styles have been getting in my way.

**Comment by jesuismaxime** [comment]
I think that this need to be a prop to wether or not use the inline-style (and I guess that most of the time that will be turn off). ie. `:inline="false"` or something else.
